src/context/db.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { createContext, useContext, useState } from "react";
+import { RxDatabase } from "rxdb";
+
+export const DbContext = createContext({});
+
+export const useDbContext = (): { db: RxDatabase; setDb: Function } =>
+  useContext(DbContext) as any;
```

Comment:
I noticed the `any` here, how strict do we intend to be with this 😊

Reply:
fixed